DesignSafe-CI/portal
# task/DES-2638 - Recon Portal Header Update PART 2

## Overview: ##
Received feedback on part one and made adjustments accordingly. Logo now takes up less width and text is smaller.


## PR Status: ##


## Related Jira tickets: ##

* [DES-2638](https://tacc-main.atlassian.net/browse/DES-2638)

## Summary of Changes: ##
Making logo's width smaller also makes its height smaller.

## Testing Steps: ##
1.  Go to https://designsafe.dev/recon-portal/ and make sure it looks like the screenshot below.
2. Play with window size and make sure the layout is responsive
3. Do a general regression test (click around to make sure everything works).

## UI Photos:
![Screenshot 2024-05-08 at 11 32 12 AM](https://github.com/DesignSafe-CI/portal/assets/7462330/4b6e828c-6fa2-413d-b0d1-9c67f93d6714)


## Notes: ##
